pkg/distributor/distributor_test.go

```diff
@@ -197,7 +197,7 @@ func TestDistributor_Push(t *testing.T) {
 			happyIngesters:       3,
 			samples:              samplesIn{num: 25, startTimestampMs: 123456789000},
 			metadata:             5,
-			expectedGRPCError:    status.New(codes.ResourceExhausted, newIngestionRateLimitedError(20, 20).Error()),
+			expectedGRPCError:    status.New(codes.ResourceExhausted, newIngestionBurstSizeLimitedError(20, 55).Error()),
```

Comment:
Why is it 55?  I see 25 samples and 5 metadata.

Reply:
Checking.

Reply:
@bboreham It's implicitly 50 samples, because each test case makes `samples` write requests with one sample and one histogram.